Tools/CISupport/ews-build/steps.py

```diff
@@ -1720,23 +1720,29 @@ class CloseBug(buildstep.BuildStep, BugzillaMixin):
     flunkOnFailure = False
     haltOnFailure = False
 
-    def start(self):
-        self.bug_id = self.getProperty('bug_id', '')
-        if not self.bug_id:
-            self._addToLog('stdio', 'bug_id build property not found.\n')
-            self.descriptionDone = 'No bug id found'
-            self.finished(FAILURE)
-            return None
+    def __init__(self, **kwargs):
```

Comment:
seems like this __init__ can be removed, can initialize bug_id as class variable or can handle it appropriately in getResultsummary as well. But this is fine as well.